### PR TITLE
chore: Update dependencies with issues

### PR DESCRIPTION
**Pull request by jannfis**
This PR updates

* `react-form` to version 2.16.2
* `dagre` to version 0.8.5
* `webpack-cli` to version 3.3.11

as well as their direct dependencies.

Separate PR for argo-cd follows after this has been merge.

### CODE REVIEW

**Comment by jannfis** [comment]
@alexec @simster7 These are fixes for issues reported by Snyk, please review and merge if appropriate.

**Comment by alexec** [comment]
Must test by running Argo WFs.

File: package.json [hunk 1]
```diff
@@ -17,19 +17,20 @@
         "@fortawesome/fontawesome-free": "^5.8.1",
         "@tippy.js/react": "^2.1.2",
         "@types/react-autocomplete": "^1.8.5",
-        "@types/react-form": "^2.16.1",
+        "@types/react-form": "2.16.2",
         "@types/react-helmet": "^5.0.8",
         "classnames": "^2.2.5",
         "dagre": "^0.8.2",
```
Comment (alexec) [inline package.json:23]: can we remove this? less dependencies is easier to manage

**Comment by alexec** [comment]
```
(master)]$ snyk test

Testing /Users/acollins8/argo-ui...

Tested 287 dependencies for known issues, found 10 issues, 69 vulnerable paths.


Patchable issues:

  Patch available for lodash@4.17.10
  ✗ Prototype Pollution (new) [Medium Severity][https://snyk.io/vuln/SNYK-JS-LODASH-567746] in lodash@4.17.10
    introduced by react-form@2.16.0 > react-redux@5.0.7 > lodash@4.17.10 and 3 other path(s)


Issues with no direct upgrade or patch:
  ✗ Cross-site Scripting (XSS) [Medium Severity][https://snyk.io/vuln/SNYK-JS-JQUERY-565129] in jquery@3.4.1
    introduced by foundation-sites@6.4.3 > jquery@3.4.1
  This issue was fixed in versions: 3.5.0
  ✗ Cross-site Scripting (XSS) [Medium Severity][https://snyk.io/vuln/SNYK-JS-JQUERY-567880] in jquery@3.4.1
    introduced by foundation-sites@6.4.3 > jquery@3.4.1
  This issue was fixed in versions: 3.5.0
  ✗ Information Exposure [Low Severity][https://snyk.io/vuln/SNYK-JS-KINDOF-537849] in kind-of@6.0.2
    introduced by webpack-cli@3.3.5 > findup-sync@3.0.0 > micromatch@3.1.10 > kind-of@6.0.2 and 44 other path(s)
  This issue was fixed in versions: 6.0.3
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-LODASH-450202] in lodash@4.17.10
    introduced by react-form@2.16.0 > react-redux@5.0.7 > lodash@4.17.10 and 1 other path(s)
  This issue was fixed in versions: 4.17.12
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-LODASH-73638] in lodash@4.17.10
    introduced by react-form@2.16.0 > react-redux@5.0.7 > lodash@4.17.10 and 1 other path(s)
  This issue was fixed in versions: 4.17.11
  ✗ Regular Expression Denial of Service (ReDoS) [Medium Severity][https://snyk.io/vuln/SNYK-JS-LODASH-73639] in lodash@4.17.10
    introduced by react-form@2.16.0 > react-redux@5.0.7 > lodash@4.17.10 and 1 other path(s)
  This issue was fixed in versions: 4.17.11
  ✗ Prototype Pollution [Medium Severity][https://snyk.io/vuln/SNYK-JS-MINIMIST-559764] in minimist@1.2.0
    introduced by webpack-cli@3.3.5 > loader-utils@1.2.3 > json5@1.0.1 > minimist@1.2.0
  This issue was fixed in versions: 0.2.1, 1.2.3
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-SETVALUE-450213] in set-value@0.4.3
    introduced by webpack-cli@3.3.5 > findup-sync@3.0.0 > micromatch@3.1.10 > snapdragon@0.8.2 > base@0.11.2 > cache-base@1.0.1 > set-value@2.0.0 and 9 other path(s)
  This issue was fixed in versions: 2.0.1, 3.0.1
  ✗ Prototype Pollution [Medium Severity][https://snyk.io/vuln/SNYK-JS-YARGSPARSER-560381] in yargs-parser@13.1.1
    introduced by webpack-cli@3.3.5 > yargs@13.2.4 > yargs-parser@13.1.1
  This issue was fixed in versions: 13.1.2, 15.0.1, 18.1.1
```

**Pull request closed without merging**